### PR TITLE
add_rstl

### PR DESCRIPTION
**Pull request by rpanai**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/typish/meta.yaml [hunk 1]
```diff
@@ -1,42 +0,0 @@
-{% set name = "typish" %}
```
Comment (scopatz) [inline recipes/typish/meta.yaml:1]: Please do not touch files outside of the PRs own recipe

Comment (rpanai) [reply recipes/typish/meta.yaml:1]: I supposed I had to ship just `example` and my own recipe only.

Comment (scopatz) [reply recipes/typish/meta.yaml:1]: No the other recipes are staged.

Comment (rpanai) [reply recipes/typish/meta.yaml:1]: @scopatz Any idea why it's failing on Linux? Do you want me to close this PR?

**Comment by scopatz** [comment]
Maybe try merging from master?

**Pull request closed without merging**